src/components/pages/ParentFamily/ParentFamilyHome.js

```diff
@@ -1,9 +1,34 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { useParams } from 'react-router';
 import Banner from '../../common/Banner';
 import ParentSidebar from '../ParentHome/ParentSidebar';
+import CreateNewStudent from './CreateNewStudent';
+import CurrentCoursesDetails from './CurrentCoursesDetails';
 import '../../../styles/ParentStyles/index.less';
 
 const ParentFamilyHome = () => {
+  const [studentInfo, setStudentInfo] = useState(null);
+  const [modal, setModal] = useState(false);
+  const [modal2, setModal2] = useState(false);
+
+  useEffect(() => {
+    const token = JSON.parse(localStorage.getItem(`okta-token-storage`));
+    const config = {
+      headers: { Authorization: `Bearer ${token.idToken.value}` },
+    };
+    axios
+      .get(`https://coder-heroes-api.herokuapp.com/parent/1/children`, config)
+      .then(res => {
+        const familyData = res.data;
+        setStudentInfo(familyData);
+        console.log(res.data);
+      })
+      .catch(err => {
+        console.log(`error fetching axios call`);
```

Comment:
would it be better to take advantage of whatever is being returned in the "err" object or do you feel hard-coding the error is better here?

Reply:
I actually tend to agree that passing the values contained in that error would be better error handling - let's look out for things like this moving forward!